docs/API.md

```diff
@@ -28,7 +28,7 @@ The following table shows the current [JSON RPC API](https://eth.wiki/json-rpc/A
 |                                         |              |                                            |
 | eth_getTransactionByHash                | -            | not yet implemented                        |
 | eth_getTransactionByBlockHashAndIndex   | -            | not yet implemented                        |
-| eth_getTransactionByBlockNumberAndIndex | -            | not yet implemented                        |
+| eth_getTransactionByBlockNumberAndIndex | Yes          |                                            |
```

Comment:
Please add in this PR also `eth_getUncleCountByBlockNumber` implementation because it's closely related to this one

Reply:
oopps.